quickstarts/android/01-login.md

```diff
@@ -0,0 +1,162 @@
+            android:screenOrientation="portrait"
+            android:theme="@style/Lock.Theme">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="YOUR_AUTH0_DOMAIN"
+                    android:pathPrefix="/android/YOUR_APP_PACKAGE_NAME/callback"
+                    android:scheme="https" />
+            </intent-filter>
+        </activity>
+        
+        <activity android:name="com.auth0.android.lock.provider.WebViewActivity"></activity>
+
+Also, you need to add the following permissions inside the :
+        
+	<uses-permission android:name="android.permission.INTERNET" />
+	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	
+At last, don't forget to declare 	the activities you're using in the Manifest.
+	
+	
+	<activity android:name=".activities.LockActivity"/>
+	<activity android:name=".activities.MainActivity"/>
+	
+	
+> It's suggested to add both the Auth0DomainID and Auth0ClientID to the Strings.xml file rather than hardcode them in the manifest.
+        
+### 3. Implement the Login
+
+At this point, you're all set to implement the Login in any activity you want. 
+
+First, add on the onCreate method:
+
+```android
+Auth0 auth0 = new Auth0(getString(R.string.auth0_client_id), getString(R.string.auth0_domain));
+            this.lock = Lock.newBuilder(auth0, callback)
+                    // Add parameters to the Lock Builder
+                    .build();
+```
+
+Second, add on the onDestroy method:
+
+```android
+ protected void onDestroy() {
+            super.onDestroy();
+            // Your own Activity code
+            lock.onDestroy(this);
+            lock = null;
+        }
+```
+Third, add the authentication callback, inside your activity:
+
+```
+private LockCallback callback = new AuthenticationCallback() {
+            @Override
+            public void onAuthentication(Credentials credentials) {
+				// Login Success response
+            }
+
+            @Override
+            public void onCanceled() {
+				// Login Cancelled response
+            }
+
+            @Override
+            public void onError(LockException error){
+				// Login Error response
+            }
+        };
+```
+
+Finally, whenever you want to start the login widget, call:
+
+```
+            startActivity(this.lock.newIntent(this));
+
+```
+
+[![Lock.png](/media/articles/libraries/lock-android/login.png)]()
+
+> If you need in depth configuration, check more information on [Lock Builder](https://auth0.com/docs/libraries/lock-android#lock-builder)
+
+> There are multiple ways of implementing the login dialog. What you see above is the default widget; however, if you want, you can use [your own UI](02-customlogin.md).
```

Comment:
It should be `02-custom-login.md`, (`03-session-handling.md`, `04-user-profile.md` and so on...) make sure you rename your files and update every appearance accordingly.